tools/scylla-nodetool.cc

```diff
@@ -3290,6 +3326,7 @@ Fore more information, see: https://opensource.docs.scylladb.com/stable/operatin
                 },
                 {
                     typed_option<sstring>("keyspace", "The keyspace name", 1),
+                    typed_option<sstring>("table", "The column family name", 1),
```

Comment:
Let's use the new terminology and also mention tablets: "The table name (needed to display load information, if the keyspace uses tablets)"

Reply:
Fixed